test/unit/app/controllers/swaps-test.js

```diff
@@ -350,16 +356,417 @@ describe('SwapsController', function () {
     })
 
     describe('_findTopQuoteAndCalculateSavings', function () {
+      let swapsController2
```

Comment:
Removed in e4bdc815e (and in fact, no such variable needs to be defined in this scope, because the `swapsController` variable defined in the parent scope is set in the respective `beforeEach`)